client/src/App.jsx

```diff
@@ -1,10 +1,21 @@
 import Intro from "./pages/Intro";
+import LangSelection from "./pages/LangSelection";
+import Interest from "./pages/Interest";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
-import Occupation from "./pages/Occupation";
```

Comment:
don't remove, this is not your code, unsless you KNOW that its not used anymore

Reply:
understood, I reverted this line of code